package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "babel-eslint",
-  "version": "11.0.0-beta.0",
+  "version": "11.0.0-beta.1",
```

Comment:
I manually bumped this version because
1) 11.0.0-beta.1 has been published from babel repo
2) the old publish process in this repo still works if it starts from `beta.1`